crates/ruff/src/rules/pylint/snapshots/ruff__rules__pylint__tests__PLW0406_import_self.py.snap

```diff
@@ -0,0 +1,10 @@
+---
+source: crates/ruff/src/rules/pylint/mod.rs
+---
+import_self.py:1:1: PLW0406 Module import itself
```

Comment:
```suggestion
import_self.py:1:1: PLW0406 Module imports itself
```

Reply:
I would favour this too, but this is the message that pylint emits, should we mirror their behaviour? I don't mind if we don't, just got to do what makes sense.

Reply:
We often deviate from upstream when it comes to the messages we omit, if we think we can make them clearer.